app/main.ipynb

```diff
@@ -336,43 +440,1073 @@
+       " 878,\n",
+       " 879,\n",
+       " 880,\n",
+       " 881,\n",
+       " 882,\n",
+       " 883,\n",
+       " 884,\n",
+       " 885,\n",
+       " 886,\n",
+       " 887,\n",
+       " 888,\n",
+       " 889,\n",
+       " 890,\n",
+       " 891,\n",
+       " 892,\n",
+       " 893,\n",
+       " 894,\n",
+       " 895,\n",
+       " 896,\n",
+       " 897,\n",
+       " 898,\n",
+       " 899,\n",
+       " 900,\n",
+       " 901,\n",
+       " 902,\n",
+       " 903,\n",
+       " 904,\n",
+       " 905,\n",
+       " 906,\n",
+       " 907,\n",
+       " 908,\n",
+       " 909,\n",
+       " 910,\n",
+       " 911,\n",
+       " 912,\n",
+       " 913,\n",
+       " 914,\n",
+       " 915,\n",
+       " 916,\n",
+       " 917,\n",
+       " 918,\n",
+       " 919,\n",
+       " 920,\n",
+       " 921,\n",
+       " 922,\n",
+       " 923,\n",
+       " 924,\n",
+       " 925,\n",
+       " 926,\n",
+       " 927,\n",
+       " 928,\n",
+       " 929,\n",
+       " 930,\n",
+       " 931,\n",
+       " 932,\n",
+       " 933,\n",
+       " 934,\n",
+       " 935,\n",
+       " 936,\n",
+       " 937,\n",
+       " 938,\n",
+       " 939,\n",
+       " 940,\n",
+       " 941,\n",
+       " 942,\n",
+       " 943,\n",
+       " 944,\n",
+       " 945,\n",
+       " 946,\n",
+       " 947,\n",
+       " 948,\n",
+       " 949,\n",
+       " 950,\n",
+       " 951,\n",
+       " 952,\n",
+       " 953,\n",
+       " 954,\n",
+       " 955,\n",
+       " 956,\n",
+       " 957,\n",
+       " 958,\n",
+       " 959,\n",
+       " 960,\n",
+       " 961,\n",
+       " 962,\n",
+       " 963,\n",
+       " 964,\n",
+       " 965,\n",
+       " 966,\n",
+       " 967,\n",
+       " 968,\n",
+       " 969,\n",
+       " 970,\n",
+       " 971,\n",
+       " 972,\n",
+       " 973,\n",
+       " 974,\n",
+       " 975,\n",
+       " 976,\n",
+       " 977,\n",
+       " 978,\n",
+       " 979,\n",
+       " 980,\n",
+       " 981,\n",
+       " 982,\n",
+       " 983,\n",
+       " 984,\n",
+       " 985,\n",
+       " 986,\n",
+       " 987,\n",
+       " 988,\n",
+       " 989,\n",
+       " 990,\n",
+       " 991,\n",
+       " 992,\n",
+       " 993,\n",
+       " 994,\n",
+       " 995,\n",
+       " 996,\n",
+       " 997,\n",
+       " 998,\n",
+       " 999,\n",
+       " ...]"
+      ]
      },
-     "execution_count": 18,
+     "execution_count": 27,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
-   "source": [
-    "# Python code\n",
-    "[i for i in range(1000000)]"
-   ],
-   "metadata": {
-    "collapsed": false,
-    "ExecuteTime": {
-     "end_time": "2023-07-03T13:16:56.167401Z",
-     "start_time": "2023-07-03T13:16:56.125237Z"
-    }
-   }
+   "execution_count": 27
   },
   {
    "cell_type": "code",
-   "execution_count": 19,
-   "outputs": [],
    "source": [
-    "# write your numpy code here and estimate the time of execution"
+    "arr_np = np.arange(1000000)\n",
+    "%time arr_np"
```

Comment:
The use of `np.arange` to create an array of numbers from 0 to 999999 is correct for the NumPy implementation.